### PR TITLE
[SETI-1156] Add Lambda helper

### PR DESCRIPTION
**Pull request by marcinwyszynski**
@deliveroo/seti PTAL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +65,18 @@ In order to use `ensure_head`, just add it as a step to your CircleCI config fil
...
+## Custom helper: `push_lambda`
+
+`push_lambda` is a helper to push an externally generated zip file to S3 and use it to update the source code of a Lambda function. It assumes that `AWS_S3_BUCKET_NAME`, `AWS_S3_OBJECT_KEY` and `AWS_REGION` environment variables are set - either directly or indirectly (see `print_env`),
+
+In order to use `push_lambda`, just add it as a step to your CircleCI config file:
+
+```yaml
+- run:
+    name: Update Lambda function
+    command: push_lambda $function_name
```
Comment (bencord0) [inline README.md]: Where do we get `$function_name` from? Or does it need to be hard coded on a per app basis?

Comment (marcinwyszynski) [reply README.md]: It's a positional argument.

Comment (marcinwyszynski) [reply README.md]: Yes, hardcoded in Circle.

File: scripts/push_lambda.sh [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+fi
+
+FUNCTION_NAME=$1
+
+`print_env ${TARGET}`
+
+set -ex
+
+# Copy source code to S3.
+ARTEFACT=workspace/paperwatch-${CIRCLE_SHA1}.zip
```
Comment (buraks) [inline scripts/push_lambda.sh]: Is this supposed to be "artifact"?

Comment (marcinwyszynski) [reply scripts/push_lambda.sh]: Isn't is UK vs US spelling?

**Pull request merged**